apps/web/styles/globals.css

```diff
@@ -1,3 +1,14 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+  body {
+    @apply h-full;
+  }
+
+  .template-action-button {
```

Comment:
This should be a component. All reusability in React should always be driven by components.